src/Statement/Statement.php

```diff
@@ -45,12 +45,40 @@ class Statement extends Claim {
 	 *
 	 * @param Claim $claim
 	 * @param ReferenceList|null $references
+	 * or
+	 * @param Snak $mainSnak
+	 * @param Snaks|null $qualifiers
+	 * @param ReferenceList|null $references
+	 * @param string|null $guid
```

Comment:
I'm afraid no tool is able to understand this, but I also don't see a better way. This is meant to be temporary, right?